### PR TITLE
Remove a UUID test

### PR DESCRIPTION
**Pull request by marakiis**
NetworkInterface.getNetworkIterfaces().nextElement().macAddress() and
UUID.macAddress() gives different results

### CODE REVIEW

File: src/test/java/org/waarp/common/utility/test/UUIDTest.java [hunk 1]
```diff
@@ -132,13 +132,19 @@ public void testDateField() {
 
     @Test
     public void testMacAddressField() throws Exception {
+        /*
+	 * This test is void: 
+	 * NetworkInterface...getHardwareAddress and UUID.macAddress() have different results
```
Comment (fredericBregier) [inline src/test/java/org/waarp/common/utility/test/UUIDTest.java:137]: Yes there are different since we take only the first 6 bytes, still the comparison is a correct way to ensure that the maxAddress field from UUID is really a part of the original networkInterface (or configuration field).
However the test is incomplete since we should take the result from 

    private static byte[] macAddress()

By changing it to package protected such that the test could call it.

So chaning the signature of this private static method to something like:

    static byte[] macAddress()

should be enough to fix this issue.

**Review by fredericBregier — COMMENTED** [review]
Incorrect merge commit since this test is not void !

**Review by bcarlin — COMMENTED** [review]
Indeed,
The problem with this test is that it takes the first found interface as a reference, which *may* be the one that is used for UUID generation.

@fredericBregier what is the rational for this implementation ? (i.e. why not use the stdlib java.util.UUID ?)

**Comment by fredericBregier** [comment]
@bcarlin The standard UUID is not a GUUID (Global UUID). This one tends to be a global one. It is largely inspired from a previous implementation in Netty and reused almost in Vitam as is. As several servers communicate to each other and this (G)UUID  is supposed to be unique, whatever the servers are, if we used UUID we could have a lot of collision, therefore having issue to follow one specific transfer (since 2 servers can generate the very same UUID using the standard implementation). This is a usual case in distributed implementation. (note, in Vitam we use a more "rich" implementation to ensure more robustness on this collision less property but the current implementation should be enough for Waarp since there are no millions GUUID per seconds).

The test should be changed by changing the signature of `macAddress()` to remove `private` in order to be package protected. Then replace the following in the test line 135/140 (and removing also the below test on `mac null or less than 6`:

     byte[] mac = NetworkInterface.getNetworkInterfaces().nextElement().getHardwareAddress();

by

     byte[] mac = UUID.macAdress(); // should be visible then

This should make the test more robust and correct (according to the implementation), while still masking this method.
